shell/platform/fuchsia/flutter/accessibility_bridge.h

```diff
@@ -105,28 +111,22 @@ class AccessibilityBridge
           OnAccessibilityActionRequestedCallback callback) override;
 
  private:
+  // Fuchsia's default root semantic node id.
+  static constexpr int32_t kRootNodeId = 0;
+
   // Holds only the fields we need for hit testing.
   // In particular, it adds a screen_rect field to flutter::SemanticsNode.
   struct SemanticsNode {
-    int32_t id;
-    int32_t flags;
-    bool is_focusable;
-    SkRect rect;
+    flutter::SemanticsNode data;
```

Comment:
This will potentially keep more data in memory than necessary.

Reply:
answered in the thread above.